package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "pgcc",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "format": "prettier --write ."
+  },
+  "dependencies": {
+    "@chakra-ui/react": "^1.8.6",
+    "@emotion/react": "^11.8.2",
+    "@emotion/styled": "^11.8.1",
+    "axios": "^0.26.1",
+    "commitizen": "^4.2.4",
+    "framer-motion": "^6.2.8",
+    "git-cz": "^4.8.0",
```

Acredito que essas duas dependências sejam dependências de desenvolvimento, caso positivo, elas deveriam estar listadas nas devDependencies.

```suggestion
    "framer-motion": "^6.2.8",
```